site-src/implementations.md

```diff
@@ -392,6 +394,21 @@ If you have any suggestions or experience issues with NGINX Gateway Fabric, plea
 [nginx-disc-new]:https://github.com/nginxinc/nginx-gateway-fabric/discussions/new
 [nginx-slack]:https://nginxcommunity.slack.com/channels/nginx-gateway-fabric
 
+
+### ngrok Kubernetes Operator
+
+[ngrok Kubernetes Operator][ngrok-k8s-operator] provides an implementation of the Gateway API that uses [ngrok's ingress-as-a-service][ngrok]. This project uses the Gateway API to support routing traffic from ngrok's global network to applications running on Kubernetes clusters. This easily adds the benefits of ngrok, like security, network policy, and a global presence with the simplicity of cloud service. The operator contains both a Gateway API implementation as well as a controller using Kubernetes Ingress. The Gateway API implementation is currently under development and supports only the Gateway, GatewayClass and HTTPRoute. As the TLSRoute and TCPRoute move from experimental to stable, they will also be implemented.
```

Comment:
So it's possible I'm missing something, so please do correct me if I am: the title here is `ngrok Kubernetes Operator` however the links provided here are all to what appears to be called the `ngrok Kubernetes Ingress Controller`. I looked at the repository you provided and the documentation on your site and everything seems to be `Ingress Controller`. I couldn't find any relevant pages when searching "operator" in your docs site, and I also searched all your issues and PRs for the term "operator" and didn't find anything that stood out to me. The `README.md` for the repository also didn't have "operator" in it. Was this a typo? Was this meant to say `ngrok Kubernetes Ingress Controller`? OR is it the wrong link, are we missing a link to the right repository?